bootstrap/autoload.php

```diff
@@ -1,60 +1,72 @@
 <?php
 
-/*----------------------------------------------------*/
-// Paths
-/*----------------------------------------------------*/
 $rootPath = dirname(__DIR__);
-$webrootPath = $rootPath.DS.'htdocs';
 
 /*----------------------------------------------------*/
 // Composer autoload
 /*----------------------------------------------------*/
 if (file_exists($autoload = $rootPath.DS.'vendor'.DS.'autoload.php')) {
-    require_once $autoload;
+    die("Couldn't load the Composer autoload file.");
 }
+require_once $autoload;
```

Comment:
Yeah you're right but for some reason it worked this way anyway so I'll take a closer look when I get home. Thanks Rhinni